src/frontend/js/app/nginx/proxy/location-item.ejs

```diff
@@ -0,0 +1,63 @@
+<div class="location-block card">
+    <div class="card-body">
+        <div class="row">
+            <div class="col-sm-12">
+                <div class="form-group">
+                    <label class="form-label"><%- i18n('locations', 'location_label') %> <span class="form-required">*</span></label>
+                    <div class="row gutter-xs">
+                        <div class="col">
+                            <div class="input-group">
+                                <span class="input-group-prepend">
+                                    <span class="input-group-text">location</span>
+                                </span>
+                                <input type="text" name="path" class="form-control model" value="<%- path %>" placeholder="<%- i18n('locations', 'path') %>" required>
+                            </div>
+                        </div>
+                        <div class="col-auto">
+                            <div class="selectgroup">
+                                <label class="selectgroup-item">
+                                    <input type="checkbox" class="selectgroup-input">
+                                    <span class="selectgroup-button">
+                                        <i class="fe fe-eye"></i>
```

Comment:
Small improvement, the icon should be `fe-settings` to be consistent with the advanced settings for the main host